src/jdk.compiler/share/classes/com/sun/tools/javac/tree/JCTree.java

```diff
@@ -3570,6 +3604,7 @@ public abstract static class Visitor {
         public void visitNewArray(JCNewArray that)           { visitTree(that); }
         public void visitLambda(JCLambda that)               { visitTree(that); }
         public void visitParens(JCParens that)               { visitTree(that); }
+        public void visitReconstruction(JCDerivedInstance that) { visitTree(that); }
```

Comment:
Maybe `visitDerivedInstance` to be in sync with the `JCDerivedInstance` world?

Reply:
Done. Thanks!